### PR TITLE
Update guidance on replaceable

### PR DESCRIPTION
**Pull request by dariavladykina**
This update is about #7

### CODE REVIEW

**Review by jfaltenbacher — APPROVED** [review]
LGTM

File: xml/docu_styleguide.taglist.xml [hunk 1]
```diff
@@ -25,7 +25,7 @@
   respective category.
  </para>
  <para>
-  With Geekodoc 2, it is also possible to use the Internazionalization Tag Set
+  With GeekoDoc 2, it is also possible to use the Internazionalization Tag Set
```
Comment (tomschr) [inline xml/docu_styleguide.taglist.xml:28]: Typo:

```suggestion
  With GeekoDoc 2, it is also possible to use the Internationalization Tag Set
```

**Review by tomschr — CHANGES_REQUESTED** [review]
@dariavladykina All fine with only one minor typo. :wink:

File: xml/docu_styleguide.taglist.xml (hunk 1 above)
Comment (dariavladykina) [reply xml/docu_styleguide.taglist.xml:28]: Completely missed this one - thanks so much @tomschr!

**Review by tomschr — APPROVED** [review]
Looks fine, Daria! :+1:

**Pull request merged**